app/views/drivers/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1>Our Awesome Drivers!</h1>
+
+<section>
+<ul class='driver-view'>
+  <li class='columns'>
+    <a class='name'>Name</a> <a>Avg Rating</a>
+  </li>
+  <% @drivers.each do |driver| %>
+
+    <li>
+      <img src="http://placecage.com/g/100/100" alt="photo"/>
```

Comment:
Just a small note, it's human nature to want to click on a user's picture, so making the picture a link as well, would be good.